From b446f6a81931cd036396f5b7c069ce1b8087dc04 Mon Sep 17 00:00:00 2001
From: Shaohui Xie <Shaohui.Xie@freescale.com>
Date: Wed, 12 Mar 2014 14:40:01 +0800
Subject: [PATCH 595/987] fman/dts: fix compatible of mdio node for fman v3

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Mdio node's compatible for each MAC should be "fsl,fman-memac-mdio"
instead of "fsl,fman-memac-tbi" in fman v3.

Signed-off-by: Shaohui Xie <Shaohui.Xie@freescale.com>
Change-Id: I848527f67b4e5d033fcfbb739d78341de576f6d0
Reviewed-on: http://git.am.freescale.net:8181/9661
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Tiefei Zang <tie-fei.zang@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
---
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-0.dtsi | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-1.dtsi | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-0.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-1.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-2.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-3.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-4.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-5.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-0.dtsi | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-1.dtsi | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-0.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-1.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-2.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-3.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-4.dtsi  | 2 +-
 arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-5.dtsi  | 2 +-
 16 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-0.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-0.dtsi
index 72c306c..0a6558f 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-0.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-0.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@f1000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xf1000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-1.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-1.dtsi
index c53dadc..bdc3614 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-1.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-10g-1.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@f3000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xf3000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-0.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-0.dtsi
index 5d34959..c8befe8 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-0.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-0.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@e1000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe1000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-1.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-1.dtsi
index 39620a3..156bed7 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-1.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-1.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@e3000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe3000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-2.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-2.dtsi
index 9c1fb1a..7b2eab2 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-2.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-2.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@e5000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe5000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-3.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-3.dtsi
index 5d2ba1a..a4a82c5 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-3.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-3.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@e7000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe7000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-4.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-4.dtsi
index 4b1820b..7200e7e 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-4.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-4.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@e9000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe9000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-5.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-5.dtsi
index aa06d13..609cdeb 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-5.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-0-1g-5.dtsi
@@ -56,7 +56,7 @@ fman@400000 {
 	mdio@eb000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xeb000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-0.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-0.dtsi
index b63fb54..82f03a4 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-0.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-0.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@f1000 {
                 #address-cells = <1>;
                 #size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
                 reg = <0xf1000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-1.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-1.dtsi
index 56cb7b1..4707878 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-1.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-10g-1.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@f3000 {
                 #address-cells = <1>;
                 #size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
                 reg = <0xf3000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-0.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-0.dtsi
index 6a4fea5..33c4b66 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-0.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-0.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@e1000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe1000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-1.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-1.dtsi
index 80f0cd9..1a71586 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-1.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-1.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@e3000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe3000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-2.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-2.dtsi
index a0cbf96..01584a7 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-2.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-2.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@e5000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe5000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-3.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-3.dtsi
index 636ff3e..2ed6c85 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-3.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-3.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@e7000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe7000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-4.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-4.dtsi
index ba12e35..29ac182 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-4.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-4.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@e9000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xe9000 0x1000>;
 	};
 };
diff --git a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-5.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-5.dtsi
index c8d145e..2cdeb24 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-5.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-fman3-1-1g-5.dtsi
@@ -56,7 +56,7 @@ fman@500000 {
 	mdio@eb000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,fman-memac-tbi";
+		compatible = "fsl,fman-memac-mdio";
 		reg = <0xeb000 0x1000>;
 	};
 };
-- 
1.9.1

